tools/meson.build

```diff
@@ -17,6 +17,21 @@ executable('print-persistent-cache',
     install: false,
 )
 
+print_aggregates_sources = [
+    '../daemon/emer-aggregate-tally.c',
+    'print-aggregates.c',
+]
+
+executable('print-aggregates',
+    print_aggregates_sources,
+    dependencies: [
+        emer_required_modules,
+        emer_shared_dep,
+    ],
+    include_directories: include_directories('../daemon'),
+    install: false,
```

Comment:
It would probably be more useful to install this (perhaps under a different name) but I'd want to somehow make it clear that it's a debugging tool rather than something that can be relied upon in scripts, to reserve the right to change it at any time.